nemo/collections/nlp/modules/common/megatron/mlp.py

```diff
@@ -83,18 +83,30 @@ def __init__(
         self.dropout = dropout
         self.set_accepted_adapter_types([MLPInfusedAdapterConfig._target_])
 
-        if activation not in ['gelu', 'geglu', 'reglu', 'swiglu', 'squared-relu']:
+        if activation not in [
+            'gelu',
+            'geglu',
+            'reglu',
+            'swiglu',
+            'squared-relu',
+            'fast-geglu',
+            'fast-swiglu',
+            'fast-reglu',
+        ]:
             raise ValueError(
                 f"Activation {activation} not supported. Only gelu, geglu, reglu, swiglu, squared-relu are supported."
             )
```

Comment:
let's just call the list of supported activations something then we can we can print out the supported ones in the error message.

Reply:
Done!